scripts/snip.py

```diff
@@ -91,7 +91,7 @@
             else:
                 id = "snip_%s_%d" % (section, snipnum)
             if kind == "bash":
-                script = "\n%s() {\n" % id
+                script = "\n# shellcheck disable=SC2046\n%s() {\n" % id
```

Comment:
This is wrong. We should not be disabling any lint checks in generated bash functions. If there are lint errors in the generated snip.sh functions, it means that the command in the original index.md file is not following bash best practice. This is another benefit of the new model: we get lint checking of the commands that we are publishing in the docs.

You should fix the lint problem by changing the command in index.md and then regen.

Reply:
Done: the lint problem is fixed by changing the command in index.md and the disabling lint check has been removed.